source/includes/fact-oplog-background.rst

```diff
@@ -4,14 +4,22 @@ to the data on the destination cluster.  When operations
 that ``mongosync`` has not applied roll off the ``oplog`` 
 on the source cluster, the sync fails and ``mongosync`` exits.
 
-During the initial sync, ``mongosync`` may apply operations at a slower
-rate due to the load imposed by copying documents concurrently.
-After ``mongosync`` completes the initial sync, it applies changes 
-faster and is more likely to maintain a position in the ``oplog``
-that is close to the real-time writes occuring on the source cluster.
+Starting in version 1.5.0, {+c2c-product-name+} enables Oplog Rollover
+Resilience (ORR).  With ORR,  ``mongosync`` applies changes on the
+source cluster to the destination cluster during the initial sync. ORR
+increases the resilience of ``mongosync`` to oplog rollover but does not
+prevent rollover entirely.
+
+You might exceed the oplog window if you: 
+
+- Sync from a high write rate source cluster for an extended
+  period.
+- Pause sync for an extended period.
+
+To increase the size of the ``oplog`` on the source cluster, use
+:setting:`~replication.oplogSizeMB`. For more information, see
+:ref:`Change Oplog Size <tutorial-change-oplog-size>` and
+:ref:`Workloads that Might Requre a Large Oplog Size
```

Comment:
I think the workloads link points to the wrong page?

Reply:
It should work now that [DOCSP-35034](https://jira.mongodb.org/browse/DOCSP-35034) is resolved! (might now be visible on the staging link)